hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -2570,6 +2614,11 @@ protected S3ListResult continueListObjects(S3ListRequest request,
               OBJECT_CONTINUE_LIST_REQUEST,
               () -> {
                 if (useListV1) {
+                  if (accessPoint != null) {
+                    // AccessPoints are not compatible with V1List
+                    throw new InvalidRequestException("ListV1 is not supported by AccessPoints");
```

Comment:
do you think maybe during initialize it should just downgrade?

Reply:
Yep, good idea, upgrading it is!

Reply:
(side issue: reviewing snowball support. it's a v1 API only)

Reply:
I upgraded list to V2 in `initialize` method. I'm thinking I should make that logging more explicit there and completely remove this extra check + `throw InvalidRequestException`. Should be enough right?

Reply:
If not, remove that and skip all V1 list tests (since they're using V2 anyway)

Reply:
I see you've reverted this & letting the SDK fail it. worksforme